### ISSUE TITLE
overlap with Agile Guide...

### ISSUE THREAD

**Issue by ultrasaurus** [18F/lean-product-design#5]
We intentionally authored Agile Guide to be cross-disciplinary, so there's a lot of overlap... what do you think about doing some cross-linking?  https://github.com/18F/agile

**Comment by ultrasaurus** [comment]
addressed this in https://github.com/18F/lean-product-design/pull/6  (along with some edits to the agile guild)

**Comment by brethauer** [comment]
Yes! @ultrasaurus where do you think would be appropriate? I think of Lean UX as sort of being a layer on agile, so of course it's really necessary to understand agile first, before trying to do lean ux with it.
